docs/get-started/quickstarts/detect-people.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Detect People with a Webcam"
+linkTitle: "Detect people with a webcam"
+weight: 50
+type: "docs"
+tags: ["vision", "data", "services", "quickstart", "ml", "camera", "webcam"]
+no_list: true
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
+imageAlt: "Machine Learning"
+---
+
+Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people).
+
+## Requirements
+
+- A computer: a development machine like your laptop and optionally a single-board computer like the Raspberry Pi
+- A webcam: this could be the webcam on your laptop or any other webcam you can connect to your computer
+- An account on the [Viam app](https://app.viam.com)
+
+First, follow the [instructions to create a new machine](/cloud/machines/#add-a-new-machine) representing your computer.
```

Comment:
For these quickstarts, I think we want everything needed to complete the task presented on this page, ideally in the same format (expanders in this case), rather than linking off-page to other resources. What do you think of:

1. Convert /the `cloud/machines/#add-a-new-machine` instructions to expander, and repeat that content here (but absolutely use the same image file!)

2. Convert "Follow the setup instructions" to a second expander, saying verbatim what you already have here (App UI is separate from docs, no way around sending users there.)

3. Delete the TOC for this page that is lines 26 - 28: This section forecasts the next three steps, but the fact that they are linked (to their respective reference content) strongly suggests to the reader that these are in fact prerequisites that they must complete first, before then opening the first expander below.

Altogether, this looks to me like something resembling:

```
{{%expand "Step 1: Create a new machine" %}}
# Dupe /cloud/machines/#add-a-new-machine - it's two sentences and an image you can reuse
{{%expand "Step 2: Install `viam-server`" %}}
# Verbatim copy to what you already have
{{%expand "Step 3: Configure your webcam" %}}
{{%expand "Step 4: Deploy a person detection model" %}}
{{%expand "Step 5: Configure your detector" %}}
{{%expand "Step 6: Test person detection" %}}
```

Expanders for steps 4 -6 can then contain the link to the related reference entry for each that you previously used as the TOC: lines 26-28. Within the expander, the user may choose to click through to more info about this current expander topic (your link), or skip past and open the next one: the IA is clearer.